.pre-commit-config.yaml

```
@@ -1,7 +1,22 @@
-files: 'shapely/'
+ci:
+  autoupdate_schedule: quarterly
```

```suggestion
  autoupdate_schedule: quarterly
  autofix_prs: false
```

That still uses the pre-commit.ci to regularly update the versions here, without the pushing to other PRs. That would personally be my preference.

---

Mind you, these configurations are dormant for now, unless we later decide to switch service to pre-commit.ci

---

Ah yes, but then should we then either just enable it, or otherwise remove the configuration? (or at least add a comment that it is not used?)

---

Let's remove the unused configuration